packages/test/page-object/src/globals/testHelpers/accessibility/createDirectLineForTest.js

```diff
@@ -0,0 +1,74 @@
+import createDeferredObservable from '../../../utils/createDeferredObservable';
```

Comment:
Can you leave a comment that this is for testing delayed activities for accessibility? Otherwise the reasoning for it being under `/accessibility/` is unclear.

Reply:
Moved under `/accessibility/delayActivity/`.